src/pages/Profile.js

```diff
@@ -0,0 +1,48 @@
+import "./Profile.css";
+
+export default function Profile() {
+  return (
+    <section>
+      <h2>you can create your profile here!</h2>
+      <div className="Form">
```

Comment:
I can see this class becoming confusing at some point as you'd expect to address the styling of the actual form. 
Maybe call it form-wrapper

Reply:
thanks. naming is hard 🥲